### PR TITLE
Add the snapshot configuration field

### PR DESCRIPTION
**Pull request by voscob**
### Description

In this pull request, a new configuration boolean field `snapshot` has been added, determining whether the connector will take a snapshot of the entire table before starting CDC mode. The default value is `true`. 

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/conduitio-labs/conduit-connector-stripe/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The config passed to `Configure` can contain the following fields:
 |----------------|----------------------------------------------------------------------------------------------------------------------|----------|----------------------------|
 | `secretKey`    | Stripe [secret key](https://dashboard.stripe.com/apikeys).                                                           | yes      | sk_51Kr0QrJit566F2YtZAwMlh |
 | `resourceName` | The name of Stripe resource. A list of supported resources can be found [here](models/resources/README.md).          | yes      | plan                       |
+| `snapshot`     | The field determines whether the connector will take a snapshot of the entire table before starting cdc mode.        | no       | false                      |
```
Comment (maksenius) [inline README.md:18]: please change  `entire table` to `entire resource`

File: config/config.go [hunk 2]
```diff
@@ -15,40 +15,56 @@
 package config
 
 import (
+	"fmt"
 	"strconv"
 
 	"github.com/conduitio-labs/conduit-connector-stripe/validator"
 )
 
+type SnapshotMode string
```
Comment (alyakimenko) [inline config/config.go:24]: This guy managed to survive somehow.

Comment (voscob) [reply config/config.go]: Oh no 🙀

**Review by maksenius — APPROVED** [review]

**Review by maha-hajja — APPROVED** [review]
lgtm

**Pull request merged**